### ISSUE TITLE
Error on abstract reference on literal

### ISSUE THREAD

**Issue by yuchi** [6to5/6to5#384]
```sh
$ node_modules/.bin/6to5 test/setoid.js --experimental
```

```js
"lol"::lol("lol");
```

```
/path/to/prj/node_modules/6to5/lib/6to5/util.js:243
    throw err;
          ^
TypeError: test/setoid.js: Invalid node type "Literal" passed to Scope.prototype.generateTempBasedOnNode
    at Scope.generateTempBasedOnNode (/path/to/prj/node_modules/6to5/lib/6to5/traverse/scope.js:112:11)
    at exports.CallExpression (/path/to/prj/node_modules/6to5/lib/6to5/transformation/transformers/es7-abstract-references.js:81:20)
    at Object.<anonymous> (/path/to/prj/node_modules/6to5/lib/6to5/transformation/transformer.js:63:14)
    at handle (/path/to/prj/node_modules/6to5/lib/6to5/traverse/index.js:89:33)
    at traverse (/path/to/prj/node_modules/6to5/lib/6to5/traverse/index.js:125:7)
    at handle (/path/to/prj/node_modules/6to5/lib/6to5/traverse/index.js:102:7)
    at traverse (/path/to/prj/node_modules/6to5/lib/6to5/traverse/index.js:112:9)
    at handle (/path/to/prj/node_modules/6to5/lib/6to5/traverse/index.js:102:7)
    at traverse (/path/to/prj/node_modules/6to5/lib/6to5/traverse/index.js:125:7)
    at Transformer.transform (/path/to/prj/node_modules/6to5/lib/6to5/transformation/transformer.js:69:3)
```

- - -

```sh
$ node_modules/.bin/6to5 --version
2.4.10
```

**Comment by yuchi** [comment]
Yet [in the REPL](https://6to5.org/repl.html#?code=%22lol%22%3A%3Alol(%22lol%22)%3B&experimental=true&playground=true&evaluate=false) works.

**Comment by sebmck** [comment]
Pushed a new release out about a minute ago that fixes a lot of issues that were accidently merged in `master` following `esnext` compatibility development. Fixed in 2.5.0
